scripts/MergeAnalyzerMetadata.ps1

```diff
@@ -16,32 +16,47 @@ Write-Output "Creating solution file"
 Write-Output "Restoring packages"
 & dotnet restore --nologo --verbosity q
 
-# $DotnetVersion = & dotnet --version
-$DotNetVersion = "8.0.100-preview.5.23262.6"
-
 $Artifacts = Join-Path -Path $PSScriptRoot -ChildPath "\..\artifacts" -Resolve
 $DiagToolPath = "$Artifacts\bin\DiagConfig\Debug\net8.0\*"
-$Command = "$Artifacts\bin\DiagConfig\Debug\net8.0\DiagConfig.exe"
 $Diags = (Resolve-Path $PSScriptRoot).Path + "\..\eng\Diags"
-#$DotnetPath = Split-Path -Path (get-command dotnet.exe).Path -Parent
-$DotnetPath = "C:\Program Files\dotnet"
-$RoslynPath = "$DotnetPath\sdk\$DotnetVersion\Roslyn\bincore"
-$CodeStylePath = "$DotnetPath\sdk\$DotnetVersion\Sdks\Microsoft.NET.Sdk\codestyle\cs"
-$NetAnalyzersPath = "$DotnetPath\sdk\$DotnetVersion\Sdks\Microsoft.NET.Sdk\analyzers"
-$PackagePath = Join-Path -Path $Home -ChildPath ".nuget/packages" -Resolve
-$AspNetCorePath = Join-Path -Path $DotNetPath -ChildPath packs\Microsoft.AspNetCore.App.Ref\7.0.4\analyzers\dotnet\cs
-
-Write-Output "Processing analyzer assemblies"
 
-& $Command $Diags analyzer merge $Artifacts\bin\Microsoft.Analyzers.Extra\Debug\netstandard2.0\Microsoft.Analyzers.Extra.dll
-& $Command $Diags analyzer merge $Artifacts\bin\Microsoft.Analyzers.Local\Debug\netstandard2.0\Microsoft.Analyzers.Local.dll
-& $Command $Diags analyzer merge $PackagePath\stylecop.analyzers.unstable\1.2.0.435\analyzers\dotnet\cs\StyleCop.Analyzers.dll
-& $Command $Diags analyzer merge $PackagePath\sonaranalyzer.csharp\8.52.0.60960\analyzers\SonarAnalyzer.CSharp.dll
-& $Command $Diags analyzer merge $PackagePath\microsoft.visualstudio.threading.analyzers\17.5.22\analyzers\cs\Microsoft.VisualStudio.Threading.Analyzers.dll
-& $Command $Diags analyzer merge $PackagePath\microsoft.visualstudio.threading.analyzers\17.5.22\analyzers\cs\Microsoft.VisualStudio.Threading.Analyzers.CSharp.dll
-& $Command $Diags analyzer merge $PackagePath\xunit.analyzers\1.0.0\analyzers\dotnet\cs\xunit.analyzers.dll
+# TODO: needs to be auto-generated. This is a dump of the Analyzers items when compiling test/libraries/microsoft.aspnetcore.asyncstate
```

Comment:
We'll address this in a separate PR.

Reply:
Done in https://github.com/dotnet/extensions/pull/4247